datumaro/plugins/datumaro_format/converter.py

```diff
@@ -188,6 +190,18 @@ def _convert_caption_object(self, obj):
         })
         return converted
 
+    def _convert_cuboid_3d_object(self, obj):
+        converted = self._convert_annotation(obj)
+        converted.update({
+            'label_id': cast(obj.label, int),
+            'cuboid_3d': {
```

Comment:
What is the point in the `cuboid_3d` section? It is already an annotation.

Reply:
Yes, I agree this is superfluous. Removed.